lib/cmd_tool_next.go

```diff
@@ -0,0 +1,123 @@
+package lib
+
+import (
+	"fmt"
+	"net"
+	"github.com/spf13/pflag"
+)
+
+type CmdToolNextFlags struct {
+	Help  bool
+	Quiet bool
+}
+
+func (f *CmdToolNextFlags) Init() {
+	pflag.BoolVarP(
+		&f.Help,
+		"help", "h", false,
+		"show help.",
+	)
+	pflag.BoolVarP(
+		&f.Quiet,
+		"quiet", "q", false,
+		"quiet mode; suppress additional output.",
+	)
+}
+
+func CmdToolNext(
+	f CmdToolNextFlags,
+	args []string,
+	printHelp func(),
+) error {
+	if f.Help {
+		printHelp()
+		return nil
+	}
+
+	increment := 1
+	actionFunc := func(input string, inputType INPUT_TYPE) error {
+		switch inputType {
+		case INPUT_TYPE_IP:
+			UpdateIPAddress(input, increment)
+		default:
+			return ErrNotIP
+		}
+		return nil
+	}
+	err := GetInputFrom(args, true, true, actionFunc)
+	if err != nil {
+		fmt.Println(err)
+	}
+	
+	return nil
+}
+
+func UpdateIPAddress(input string, delta int) {
```

Comment:
This function name should be something else, it's too generic: `ipAdd` seems specific enough, with `delta` being the thing we're adding.

However it should also then go into a different file as it can be used in general, and should also be returning something rather than printing it - we can print from the outside.